apps/ia/ia-hearings-api/demo.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: helm.toolkit.fluxcd.io/v2beta1
+kind: HelmRelease
+metadata:
+  name: ia-hearings-api
+  namespace: ia
+spec:
+  releaseName: ia-hearings-api
+  values:
+    java:
+      disableTraefikTls: true
+      image: hmctspublic.azurecr.io/ia/hearings-api:prod-8553037-20220727161248
+      replicas: 2
+      autoscaling:
+        enabled: true
+        maxReplicas: 5
+        targetCPUUtilizationPercentage: 80
```

Comment:
Looks to be alot of duplicate content from base

Reply:
thanks, removed these duplicates in latest commit